OrbitCaptureClient/CaptureEventProcessor.cpp

```diff
@@ -120,6 +127,22 @@ void CaptureEventProcessor::ProcessFunctionCall(const FunctionCall& function_cal
   capture_listener_->OnTimer(timer_info);
 }
 
+void CaptureEventProcessor::ProcessIntrospectionCall(const IntrospectionCall& introspection_call) {
+  const FunctionCall& function_call = introspection_call.function_call();
+  TimerInfo timer_info;
+  timer_info.set_process_id(function_call.pid());
+  timer_info.set_thread_id(function_call.tid());
+  timer_info.set_start(function_call.begin_timestamp_ns());
+  timer_info.set_end(function_call.end_timestamp_ns());
+  timer_info.set_depth(static_cast<uint8_t>(function_call.depth()));
+  timer_info.set_function_address(0);
```

Comment:
I would add a comment why it's okay to use these values (0, -1) here.

Reply:
done